scripts/fast-reboot

```diff
@@ -477,7 +477,7 @@ debug "Stopped  syncd ..."
 # then immediately call `systemctl stop ...` to prevent the service from
 # restarting the container automatically.
 for CONTAINER_NAME in $(docker ps --format '{{.Names}}'); do
-    docker kill $CONTAINER_NAME > /dev/null
+    docker kill $CONTAINER_NAME > /dev/null || [ $? == 1 ]
```

Comment:
>docker kill $CONTAINER_NAME > /dev/null || [ $? == 1 ] [](start = 4, length = 54)

Why this line's redirect and pipe are different with L467

Reply:
Good observation. I think the ordering in L467 is incorrect.

Reply:
Yeah, we discussed this mistake and never fixed it :-) I'll add the change to review later when tested.

Reply:
Change is done. Thanks for noticing!